offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/dataresources/products/ProductsDbConnector.groovy

```diff
@@ -332,8 +332,10 @@ class ProductsDbConnector implements ArchiveProductDataSource, CreateProductData
 
       while(result.next()){
         String id = result.getString(1)
-
-        if(id) latestUniqueId = Long.parseLong(id.split('_')[1])
+        if(id) {
+          long idRunningNumber = Long.parseLong(id.split('_')[1])
+          if(idRunningNumber > latestUniqueId) latestUniqueId = Long.parseLong(id.split('_')[1])
```

Comment:
```suggestion
          if(idRunningNumber > latestUniqueId) latestUniqueId = Long.parseLong(idRunningNumber)
```

Reply:
🤦🏼‍♀️ thanks for pointing this out, that was my plan actually ..